lua/entities/cfc_shaped_charge/cl_init.lua

```diff
@@ -0,0 +1,41 @@
+include( 'shared.lua' )
+ 
+language.Add( "cfc_shaped_charge" )
+
+local explodeTime 
+
+function ENT:Initialize()
+    explodeTime = CurTime() + GetConVar( "cfc_shaped_charge_timer" ):GetInt()
```

Comment:
We can place it directly on the entity here
```suggestion
    self.explodeTime = CurTime() + GetConVar( "cfc_shaped_charge_timer" ):GetInt()
```